lib/broadway_kafka/producer.ex

```diff
@@ -507,8 +516,10 @@ defmodule BroadwayKafka.Producer do
     |> drain_after_revoke_table_name!()
     |> set_draining_after_revoke!(true)
 
-    GenStage.call(producer_pid, :drain_after_revoke, :infinity)
-    :ok
+    :telemetry.span([:broadway_kafka, :consumer_group, :assignments_revoked], %{}, fn ->
```

Comment:
I think this would be enough, right?

```elixir
    :telemetry.span([:broadway_kafka, :assignments_revoked], %{}, fn ->
```

I would also add the producer:

```suggestion
    :telemetry.span([:broadway_kafka, :assignments_revoked], %{producer: maybe_process_name}, fn ->
```